src/components/Modal/SbModal.vue

```diff
@@ -72,9 +76,25 @@ export default {
     /**
      * handler for close modal
      */
-    handleCloseModal(event) {
-      this.open = false
-      this.$emit('hide')
+    handleCloseModal() {
+      if (this.open) {
+        this.open = false
+        this.$nextTick(() => {
+          this.$emit('hide')
+        })
+      }
```

Comment:
Isn't a better choice to have a single method that deals with the opening/closing of the Modal? Like handleShowModal, and then you do everything on it?

Reply:
I don't know. This does not have a bad ideia, but, I don't see a need now.